docs/api-reference/decorators.md

```diff
@@ -544,6 +545,54 @@ class KafkaExample{
 }
 ```
 
+### Scheduled Workflow Decorators
+
+#### `@Scheduled(schedulerConfig: SchedulerConfig)` {#scheduled}
+
+Runs a workflow function on a specified schedule, with guarantees such as executing exactly once per scheduled interval.
+
+By default, the workflow is executed exactly once per scheduled interval.  This means executions will be started concurrently and can overlap, and that if the application is taken down and restarted, makeup work will be performed.  Tracking of execution history is implemented using system database records.  An workflow key (consisting of the workflow function name and scheduled time) is used to deduplicate any workflows that may inadvertently be initiated by the scheduler.
+
+The schedule is specified in a format similar to a traditional [`crontab`](https://en.wikipedia.org/wiki/Cron), with the following notes:
```

Comment:
Are those additional rules the same as [node-cron](https://www.npmjs.com/package/node-cron)? If so, let's put a reference to node-cron here.